bundles/org.eclipse.swt/Eclipse SWT PI/win32/org/eclipse/swt/internal/win32/OS.java

```diff
@@ -2709,6 +2710,8 @@ public static int HRESULT_FROM_WIN32(int x) {
 public static final native int GdiSetBatchLimit (int dwLimit);
 public static final native int GetACP ();
 public static final native long GetActiveWindow ();
+/** @param hWnd cast=(HWND) */
```

Comment:
If I read the [MSDN documentation correctly](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getancestor#parameters), the 2nd parameter could be one of `GA_PARENT` (`1`), `GA_ROOT` (`2`) or `GA_ROOT_OWNER` (`3`). I would specify this in the documentation or at least make it clear that only `GA_ROOT` is available.

Reply:
I am adding other parameters as well.

Reply:
There are methods like GetCharacterPlacement and GetDIBColorTable, where they have paramters like dwFlag or gwFlag and they are not the part of documentation. I think I should follow the exisiting convention and not mention them in the docs since there flags are avaialble above anyways with the same name.

Reply:
👍